core/src/main/scala/org/elasticmq/actor/QueueManagerActor.scala

```diff
@@ -20,9 +20,6 @@ class QueueManagerActor(nowProvider: NowProvider) extends ReplyingActor with Log
       if (queues.contains(queueData.name)) {
         logger.debug(s"Cannot create queue, as it already exists: $queueData")
         Left(new QueueAlreadyExists(queueData.name))
-      } else if (!queueData.deadLettersQueue.forall(dlq => queues.contains(dlq.name))) {
```

Comment:
so this code wasn't used? I see there's a test for that case, so I assume there's another path which returns the queue does not exist error message?

Reply:
That's right, it's currently checked at https://github.com/adamw/elasticmq/blob/master/rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/CreateQueueDirectives.scala#L40

Reply:
Ah, that's a very good outcome of the coverage changes, thanks :)